src/elements/content-sidebar/activity-feed/approval-comment-form/ApprovalCommentForm.js

```diff
@@ -250,18 +251,20 @@ class ApprovalCommentForm extends React.Component<Props, State> {
                             <FormattedMessage {...messages.atMentionTip} />
                         </aside>
                         {createTask ? (
-                            <AddApproval
-                                approvalDate={approvalDate}
-                                approvers={approvers}
-                                approverSelectorContacts={approverSelectorContacts}
-                                approverSelectorError={approverSelectorError}
-                                formatMessage={formatMessage}
-                                isAddApprovalVisible={isAddApprovalVisible}
-                                onApprovalDateChange={this.onApprovalDateChangeHandler}
-                                onApproverSelectorInput={this.handleApproverSelectorInput}
-                                onApproverSelectorRemove={this.handleApproverSelectorRemove}
-                                onApproverSelectorSelect={this.handleApproverSelectorSelect}
-                            />
+                            <FeatureFlag feature="activityFeed.tasks.createFromComment">
```

Comment:
Should we export these features (the ugly string) from constants.js?
So that they are all listed in 1 place. FEATURE_*

Reply:
Maybe, but since it's a path into an object, not sure if that would be redundant vs the object shape?

Reply:
I'm in favour of leaving as a string since it's a path. If we move to constants for paths, the composition of those has potential to get out of control.